Sources/ProjectDescription/ResourceFileElements.swift

```diff
@@ -7,10 +7,26 @@ import Foundation
 public struct ResourceFileElements: Codable, Equatable {
     /// List of resource file elements
     public let resources: [ResourceFileElement]
+    public let excluding: [Path]
 
     public init(resources: [ResourceFileElement]) {
         self.resources = resources
+        self.excluding = []
     }
+    
+    public init(resources: [ResourceFileElement], excluding: [Path]) throws {
```

Comment:
Nitpick: Instead of exposing a new init, I'd create a static method called resources:

```swift
static func resources(_ resources: [ResourceFileElement], excluding: [Path] = []) throws  -> ResourceFileElements{
}
```

That way when you use it from a target it reads better:

```swift
let target = Target(name: "MyTarget", resources: .resources(["xxx"], excluding: []))
```